Description: permit gettimeofday() in seccomp sandbox
 This permits log message generation, without which vsftpd fails completely.
Author: Robie Basak <robie.basak@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/1219857
Forwarded: to Chris Evans <scarybeasts@gmail.com> (no upstream VCS or bug tracker evident)
Last-Update: 2014-04-28

--- a/seccompsandbox.c
+++ b/seccompsandbox.c
@@ -300,6 +300,7 @@
   reject_nr(__NR_mremap, ENOSYS);
 
   /* Misc simple low-risk calls. */
+  allow_nr(__NR_gettimeofday); /* Used by logging */
   allow_nr(__NR_rt_sigreturn); /* Used to handle SIGPIPE. */
   allow_nr(__NR_restart_syscall);
   allow_nr(__NR_close);
